components/datepicker/CHANGELOG.md

```diff
@@ -0,0 +1,7 @@
+# Pending Changes
+
+## Breaking
+
+## New Features
+
+## Bug Fixes
```

Comment:
Do we intend for datepicker to be a public-facing component? We don't have supporting docs for it, and have been using it internally / developing it as such. I would argue we don't want to use a changelog here.